mbrl/models/gaussian_mlp.py

```diff
@@ -89,19 +93,27 @@ def __init__(
         self.in_size = in_size
         self.out_size = out_size
 
-        activation_cls = nn.SiLU if use_silu else nn.ReLU
+        if use_silu:
+            warnings.warn("use_silu is deprecated and will be removed.
+                          Use activation_name = 'SiLU' instead.")
+            activation_func = nn.SiLU()
+
+        if activation_name == "ReLU" and not use_silu:
+            activation_func = nn.ReLU()
+        else:
+            activation_func = self._get_activation_func(activation_name,activation_params)
```

Comment:
As mentioned above, since we are also including parameters, it's then better to replace this with `hydra.hydra.utils.instantiate(activation_fn_cfg)`, where `activation_fn_cfg` is a an optional dictionary with the target class and parameters. The default for this dictionary should be `None`, in which case the code defaults to `ReLU`.

Reply:
I don't fully understand the usage of hydra here but I will give it a shot !

Reply:
@luisenp please review the changes I have made and let me know :)

Reply:
Once the changes are agreed upon, I'll go ahead with adding tests and updating the example notebook.

Reply:
Thanks a lot for the changes. I'm busy today, but I'm planning to look at this tomorrow or Wednesday. Thanks!